### PR TITLE
Add skx2 from the older balena-jetson-skx2

### PR DESCRIPTION
**Pull request by floion**

### CODE REVIEW

File: layers/meta-balena-jetson/conf/machine/skx2.conf [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#@TYPE: Machine
+##@NAME: skx2
+##@DESCRIPTION: Machine configuration for the SKX2
+
+MACHINEOVERRIDES = "jetson-tx2:${MACHINE}"
+include conf/machine/jetson-tx2.conf
+
+PREFERRED_VERSION_linux-tegra = "4.4.38.skx2%"
```
Comment (acostach) [inline layers/meta-balena-jetson/conf/machine/skx2.conf:8]: Kernel in this repo is 4.9, can be dropped I guess.

File: skx2.coffee [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+		osx: 'https://docs.resin.io/jetson-tx2/nodejs/getting-started/#adding-your-first-device'
+		linux: 'https://docs.resin.io/jetson-tx2/nodejs/getting-started/#adding-your-first-device'
+
+	supportsBlink: false
+
+	yocto:
+		machine: 'skx2'
+		image: 'resin-image-flasher'
+		fstype: 'resinos-img'
+		version: 'yocto-thud'
```
Comment (acostach) [inline skx2.coffee:41]: warrior

File: layers/meta-balena-jetson/recipes-kernel/linux/linux-tegra_%.bbappend [hunk 3]
```diff
@@ -151,6 +151,18 @@ RESIN_CONFIGS[gasket] = " \
...
+RESIN_CONFIGS_DEPS[cdc_acm] = " \
+    CONFIG_TTY=y \
+"
+RESIN_CONFIGS[cdc_acm] = " \
+    CONFIG_USB_ACM=m \
+"
+RESIN_CONFIGS[wdm] = " \
+    CONFIG_USB_WDM=m \
+"
+
```
Comment (acostach) [inline layers/meta-balena-jetson/recipes-kernel/linux/linux-tegra_%.bbappend:165]: SRC_URI_append_skx2 = " \
        file://tegra186-tx2-cti-ASG916.dtb \
    "

    do_deploy_append_skx2() {
        cp ${WORKDIR}/tegra186-tx2-cti-ASG916.dtb "${DEPLOYDIR}"
    }

File: layers/meta-balena-jetson/conf/machine/skx2.conf (hunk 1 above)
Comment (floion) [reply layers/meta-balena-jetson/conf/machine/skx2.conf:8]: Indeed, I will remove that. Thanks

**Comment by acostach** [comment]
Closing as not needed for now

**Pull request closed without merging**